packages/core/tsconfig.json

```diff
@@ -8,6 +8,8 @@
     "noPropertyAccessFromIndexSignature": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
+    "allowSyntheticDefaultImports": true,
+    "jsx": "react-jsx",
```

Comment:
Why `core` package need jsx?

Reply:
Just checked it, it's being used for https://github.com/feedloop/batiq/blob/328eb8fb5fb018f17bc5c668358f955084b6ed4d/packages/core/src/DataProvider.tsx#L1

Reply:
How about separate DataProvider with `core` package mas @arrizalamin ?, for example:
```tsx
// @batiq/providers/src/index.tsx

export const DataProvider = () => {}

export const useData = () => {}
```

Then import it to `core` package.